samples/ChangeFeedManagement/Program.py

```diff
@@ -98,11 +98,11 @@ def run_sample():
                 if e.status_code == 409:
                     pass
                 else:
-                    raise errors.HTTPFailure(e.status_code)
+                    raise
```

Comment:
The proposed syntax will not create a new exception, but rather bubble up the current one captured in the `except` block.  This will keep the existing stack trace intact and make it easier to debug.

This is preferred to re-raising another copy of the same exception.